portfolio/src/main/webapp/pets.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Leo's  Pets</title>
```

Comment:
Note that `<title>` is the title of the tab that appears in the browser's title bar, if you want to put a title on the page, you still need to include a header in the body.